## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 459517`
Commit: `b6222779475950b9b013fc6f4840dd7429decc6d`

---
### `x86_64-linux`
<details>
  <summary>:x: 14 packages failed to build:</summary>
  <ul>
    <li>hyprlandPlugins.borders-plus-plus</li>
    <li>hyprlandPlugins.csgo-vulkan-fix</li>
    <li>hyprlandPlugins.hy3</li>
    <li>hyprlandPlugins.hypr-dynamic-cursors</li>
    <li>hyprlandPlugins.hyprbars</li>
    <li>hyprlandPlugins.hyprexpo</li>
    <li>hyprlandPlugins.hyprfocus</li>
    <li>hyprlandPlugins.hyprgrass</li>
    <li>hyprlandPlugins.hyprscrolling</li>
    <li>hyprlandPlugins.hyprspace</li>
    <li>hyprlandPlugins.hyprsplit</li>
    <li>hyprlandPlugins.hyprtrails</li>
    <li>hyprlandPlugins.hyprwinwrap</li>
    <li>hyprlandPlugins.xtra-dispatchers</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 13 packages built:</summary>
  <ul>
    <li>grimblast</li>
    <li>hdrop</li>
    <li>hyprland</li>
    <li>hyprland.dev</li>
    <li>hyprland.man</li>
    <li>hyprpanel</li>
    <li>hyprshade</li>
    <li>hyprshade.dist</li>
    <li>hyprshell</li>
    <li>hyprshot</li>
    <li>nwg-panel</li>
    <li>nwg-panel.dist</li>
    <li>xdg-desktop-portal-hyprland</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>hyprlandPlugins.borders-plus-plus</summary>
<pre>[ 40%] Building CXX object CMakeFiles/borders-plus-plus.dir/main.cpp.o
[ 60%] Building CXX object CMakeFiles/borders-plus-plus.dir/BorderppPassElement.cpp.o
[ 80%] Building CXX object CMakeFiles/borders-plus-plus.dir/borderDeco.cpp.o
In file included from /build/borders-plus-plus/globals.hpp:3,
                 from /build/borders-plus-plus/borderDeco.cpp:8:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/borders-plus-plus.dir/build.make:93: CMakeFiles/borders-plus-plus.dir/borderDeco.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
In file included from /build/borders-plus-plus/globals.hpp:3,
                 from /build/borders-plus-plus/main.cpp:12:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/borders-plus-plus.dir/build.make:121: CMakeFiles/borders-plus-plus.dir/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:87: CMakeFiles/borders-plus-plus.dir/all] Error 2
make: *** [Makefile:136: all] Error 2</pre>
</details>
<details>
<summary>hyprlandPlugins.csgo-vulkan-fix</summary>
<pre>    CMAKE_POLICY_DEFAULT_CMP0025


-- Build files have been written to: /build/csgo-vulkan-fix/build
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
build flags: -j32 SHELL=/nix/store/ciarnmsx8lvsrmdbjddpmx0pqjrm8imb-bash-5.3p3/bin/bash
[ 66%] Building CXX object CMakeFiles/csgo-vulkan-fix.dir/main.cpp.o
[ 66%] Building CXX object CMakeFiles/csgo-vulkan-fix.dir/CMakeFiles/4.1.2/CompilerIdCXX/CMakeCXXCompilerId.cpp.o
In file included from /build/csgo-vulkan-fix/globals.hpp:3,
                 from /build/csgo-vulkan-fix/main.cpp:12:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/csgo-vulkan-fix.dir/build.make:93: CMakeFiles/csgo-vulkan-fix.dir/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:87: CMakeFiles/csgo-vulkan-fix.dir/all] Error 2
make: *** [Makefile:136: all] Error 2</pre>
</details>
<details>
<summary>hyprlandPlugins.hy3</summary>
<pre>   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/hy3.dir/build.make:93: CMakeFiles/hy3.dir/src/dispatchers.cpp.o] Error 1
make[2]: *** [CMakeFiles/hy3.dir/build.make:149: CMakeFiles/hy3.dir/src/SelectionHook.cpp.o] Error 1
In file included from /build/source/src/Hy3Layout.cpp:15:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/hy3.dir/build.make:121: CMakeFiles/hy3.dir/src/Hy3Node.cpp.o] Error 1
make[2]: *** [CMakeFiles/hy3.dir/build.make:107: CMakeFiles/hy3.dir/src/Hy3Layout.cpp.o] Error 1
In file included from /build/source/src/main.cpp:3:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/hy3.dir/build.make:79: CMakeFiles/hy3.dir/src/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:87: CMakeFiles/hy3.dir/all] Error 2
make: *** [Makefile:136: all] Error 2</pre>
</details>
<details>
<summary>hyprlandPlugins.hypr-dynamic-cursors</summary>
<pre>source root is source
Running phase: patchPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;patchPhase&quot; }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;updateAutotoolsGnuConfigScriptsPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
no configure script, doing nothing
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
build flags: SHELL=/nix/store/ciarnmsx8lvsrmdbjddpmx0pqjrm8imb-bash-5.3p3/bin/bash
g++ -Wall --no-gnu-unique -fPIC -std=c++26 -g -I/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src -I/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include -I/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/protocols -I/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland -I/nix/store/zx88hmmi1an1gfyrfbzm4ga77qii30d5-pixman-0.46.4/include/pixman-1 -I/nix/store/qy6wc8j2pnsi5hp7h8xrjy85a6fzs47d-libdrm-2.4.126-dev/include -I/nix/store/qy6wc8j2pnsi5hp7h8xrjy85a6fzs47d-libdrm-2.4.126-dev/include/libdrm -c src/cursor.cpp -o out/cursor.o
In file included from src/globals.hpp:3,
                 from src/cursor.hpp:1,
                 from src/cursor.cpp:25:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:21: out/cursor.o] Error 1</pre>
</details>
<details>
<summary>hyprlandPlugins.hyprbars</summary>
<pre>make[2]: *** [CMakeFiles/hyprbars.dir/build.make:93: CMakeFiles/hyprbars.dir/barDeco.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
In file included from /build/hyprbars/globals.hpp:3,
                 from /build/hyprbars/barDeco.hpp:12,
                 from /build/hyprbars/BarPassElement.cpp:3:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/hyprbars.dir/build.make:79: CMakeFiles/hyprbars.dir/BarPassElement.cpp.o] Error 1
In file included from /build/hyprbars/globals.hpp:3,
                 from /build/hyprbars/barDeco.hpp:12,
                 from /build/hyprbars/main.cpp:13:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/hyprbars.dir/build.make:121: CMakeFiles/hyprbars.dir/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:87: CMakeFiles/hyprbars.dir/all] Error 2
make: *** [Makefile:136: all] Error 2</pre>
</details>
<details>
<summary>hyprlandPlugins.hyprexpo</summary>
<pre>      |          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/hyprexpo.dir/build.make:135: CMakeFiles/hyprexpo.dir/overview.cpp.o] Error 1
In file included from /build/hyprexpo/globals.hpp:3,
                 from /build/hyprexpo/main.cpp:16:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/hyprexpo.dir/build.make:121: CMakeFiles/hyprexpo.dir/main.cpp.o] Error 1
In file included from /build/hyprexpo/globals.hpp:3,
                 from /build/hyprexpo/overview.hpp:5,
                 from /build/hyprexpo/OverviewPassElement.cpp:3:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/hyprexpo.dir/build.make:93: CMakeFiles/hyprexpo.dir/OverviewPassElement.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:87: CMakeFiles/hyprexpo.dir/all] Error 2
make: *** [Makefile:136: all] Error 2</pre>
</details>
<details>
<summary>hyprlandPlugins.hyprfocus</summary>
<pre>    CMAKE_POLICY_DEFAULT_CMP0025


-- Build files have been written to: /build/hyprfocus/build
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
build flags: -j32 SHELL=/nix/store/ciarnmsx8lvsrmdbjddpmx0pqjrm8imb-bash-5.3p3/bin/bash
[ 33%] Building CXX object CMakeFiles/hyprfocus.dir/CMakeFiles/4.1.2/CompilerIdCXX/CMakeCXXCompilerId.cpp.o
[ 66%] Building CXX object CMakeFiles/hyprfocus.dir/main.cpp.o
In file included from /build/hyprfocus/globals.hpp:3,
                 from /build/hyprfocus/main.cpp:20:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/hyprfocus.dir/build.make:93: CMakeFiles/hyprfocus.dir/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:87: CMakeFiles/hyprfocus.dir/all] Error 2
make: *** [Makefile:136: all] Error 2</pre>
</details>
<details>
<summary>hyprlandPlugins.hyprgrass</summary>
<pre>compilation terminated.
[2/14] Compiling C++ object src/libhyprgrass.so.p/GestureManager.cpp.o
FAILED: [code=1] src/libhyprgrass.so.p/GestureManager.cpp.o 
g++ -Isrc/libhyprgrass.so.p -Isrc -I../src -I/nix/store/dkj2dq3bqascf1p1b3kbpri0p8lfrrcp-wf-touch-0-unstable-2021-03-19-dev/include -I/nix/store/45r64jsa9vajqbxnkzl4gxrcg3yvkkr0-glm-1.0.2/include -I/nix/store/zx88hmmi1an1gfyrfbzm4ga77qii30d5-pixman-0.46.4/include/pixman-1 -I/nix/store/9s4nkf07fcl0rbc4d36lda3b877cjc9w-libinput-1.29.1-dev/include -I/nix/store/465lj8xn3vp4xfand16iqlwx1kqgc4id-wayland-1.24.0-dev/include -I/nix/store/anzq0yqha04vzs47v32nisyqf9dqak9j-libxkbcommon-1.11.0-dev/include -I/nix/store/qy6wc8j2pnsi5hp7h8xrjy85a6fzs47d-libdrm-2.4.126-dev/include -I/nix/store/qy6wc8j2pnsi5hp7h8xrjy85a6fzs47d-libdrm-2.4.126-dev/include/libdrm -I/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include -I/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/protocols -I/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland -fdiagnostics-color=always -D_GLIBCXX_ASSERTIONS=1 -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=c++23 -fPIC -DWLR_USE_UNSTABLE -MD -MQ src/libhyprgrass.so.p/GestureManager.cpp.o -MF src/libhyprgrass.so.p/GestureManager.cpp.o.d -o src/libhyprgrass.so.p/GestureManager.cpp.o -c ../src/GestureManager.cpp
In file included from ../src/GestureManager.hpp:4,
                 from ../src/GestureManager.cpp:1:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
[3/14] Compiling C++ object src/gestures/libgestures.a.p/Shared.cpp.o
[4/14] Compiling C++ object src/gestures/libgestures.a.p/DragGesture.cpp.o
[5/14] Compiling C++ object src/gestures/libgestures.a.p/CompletedGesture.cpp.o
[6/14] Compiling C++ object src/gestures/libgestures.a.p/Actions.cpp.o
[7/14] Compiling C++ object src/gestures/test/test-gestures.p/MockGestureManager.cpp.o
[8/14] Compiling C++ object src/gestures/libgestures.a.p/Gestures.cpp.o
[9/14] Compiling C++ object src/libhyprgrass.so.p/VecSet.cpp.o
[10/14] Compiling C++ object src/libhyprgrass.so.p/TouchVisualizer.cpp.o
[11/14] Compiling C++ object src/gestures/test/test-gestures.p/test.cpp.o
ninja: build stopped: subcommand failed.</pre>
</details>
<details>
<summary>hyprlandPlugins.hyprscrolling</summary>
<pre>
-- Build files have been written to: /build/hyprscrolling/build
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
build flags: -j32 SHELL=/nix/store/ciarnmsx8lvsrmdbjddpmx0pqjrm8imb-bash-5.3p3/bin/bash
[ 25%] Building CXX object CMakeFiles/hyprscrolling.dir/CMakeFiles/4.1.2/CompilerIdCXX/CMakeCXXCompilerId.cpp.o
[ 50%] Building CXX object CMakeFiles/hyprscrolling.dir/Scrolling.cpp.o
[ 75%] Building CXX object CMakeFiles/hyprscrolling.dir/main.cpp.o
In file included from /build/hyprscrolling/globals.hpp:3,
                 from /build/hyprscrolling/main.cpp:19:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/hyprscrolling.dir/build.make:107: CMakeFiles/hyprscrolling.dir/main.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:87: CMakeFiles/hyprscrolling.dir/all] Error 2
make: *** [Makefile:136: all] Error 2</pre>
</details>
<details>
<summary>hyprlandPlugins.hyprspace</summary>
<pre>compilation terminated.
In file included from src/Globals.hpp:3,
                 from src/Overview.cpp:2:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
In file included from src/Globals.hpp:3,
                 from src/Render.cpp:2:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
In file included from /nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginSystem.hpp:5,
                 from src/main.cpp:1:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:7: all] Error 1</pre>
</details>
<details>
<summary>hyprlandPlugins.hyprsplit</summary>
<pre>    mandir       : /nix/store/5lzc95kirs8fjrj8yf5r2abl6ymz853v-hyprsplit-0.51.1/share/man
    prefix       : /nix/store/5lzc95kirs8fjrj8yf5r2abl6ymz853v-hyprsplit-0.51.1
    sbindir      : /nix/store/5lzc95kirs8fjrj8yf5r2abl6ymz853v-hyprsplit-0.51.1/sbin
    wrap_mode    : nodownload

Found ninja-1.13.1 at /nix/store/asj4vdlgiqpcqc45lq29r4hv4wlfbl63-ninja-1.13.1/bin/ninja
mesonConfigurePhase: enabled\ parallel\ building
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
build flags: -j32
[1/2] Compiling C++ object libhyprsplit.so.p/src_main.cpp.o
FAILED: [code=1] libhyprsplit.so.p/src_main.cpp.o 
g++ -Ilibhyprsplit.so.p -I. -I.. -I/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include -I/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/protocols -I/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland -I/nix/store/zx88hmmi1an1gfyrfbzm4ga77qii30d5-pixman-0.46.4/include/pixman-1 -I/nix/store/qy6wc8j2pnsi5hp7h8xrjy85a6fzs47d-libdrm-2.4.126-dev/include -I/nix/store/qy6wc8j2pnsi5hp7h8xrjy85a6fzs47d-libdrm-2.4.126-dev/include/libdrm -fdiagnostics-color=always -D_GLIBCXX_ASSERTIONS=1 -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=c++23 -DWLR_USE_UNSTABLE -fPIC -MD -MQ libhyprsplit.so.p/src_main.cpp.o -MF libhyprsplit.so.p/src_main.cpp.o.d -o libhyprsplit.so.p/src_main.cpp.o -c ../src/main.cpp
In file included from ../src/globals.hpp:3,
                 from ../src/main.cpp:1:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
ninja: build stopped: subcommand failed.</pre>
</details>
<details>
<summary>hyprlandPlugins.hyprtrails</summary>
<pre>      |          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/hyprtrails.dir/build.make:121: CMakeFiles/hyprtrails.dir/trail.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
In file included from /build/hyprtrails/trail.hpp:6,
                 from /build/hyprtrails/TrailPassElement.cpp:3:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/hyprtrails.dir/build.make:79: CMakeFiles/hyprtrails.dir/TrailPassElement.cpp.o] Error 1
In file included from /build/hyprtrails/globals.hpp:3,
                 from /build/hyprtrails/main.cpp:13:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/hyprtrails.dir/build.make:107: CMakeFiles/hyprtrails.dir/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:87: CMakeFiles/hyprtrails.dir/all] Error 2
make: *** [Makefile:136: all] Error 2</pre>
</details>
<details>
<summary>hyprlandPlugins.hyprwinwrap</summary>
<pre>    CMAKE_POLICY_DEFAULT_CMP0025


-- Build files have been written to: /build/hyprwinwrap/build
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
build flags: -j32 SHELL=/nix/store/ciarnmsx8lvsrmdbjddpmx0pqjrm8imb-bash-5.3p3/bin/bash
[ 66%] Building CXX object CMakeFiles/hyprwinwrap.dir/main.cpp.o
[ 66%] Building CXX object CMakeFiles/hyprwinwrap.dir/CMakeFiles/4.1.2/CompilerIdCXX/CMakeCXXCompilerId.cpp.o
In file included from /build/hyprwinwrap/globals.hpp:3,
                 from /build/hyprwinwrap/main.cpp:20:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/hyprwinwrap.dir/build.make:93: CMakeFiles/hyprwinwrap.dir/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:87: CMakeFiles/hyprwinwrap.dir/all] Error 2
make: *** [Makefile:136: all] Error 2</pre>
</details>
<details>
<summary>hyprlandPlugins.xtra-dispatchers</summary>
<pre>    CMAKE_POLICY_DEFAULT_CMP0025


-- Build files have been written to: /build/xtra-dispatchers/build
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
build flags: -j32 SHELL=/nix/store/ciarnmsx8lvsrmdbjddpmx0pqjrm8imb-bash-5.3p3/bin/bash
[ 33%] Building CXX object CMakeFiles/xtra-dispatchers.dir/CMakeFiles/4.1.2/CompilerIdCXX/CMakeCXXCompilerId.cpp.o
[ 66%] Building CXX object CMakeFiles/xtra-dispatchers.dir/main.cpp.o
In file included from /build/xtra-dispatchers/globals.hpp:3,
                 from /build/xtra-dispatchers/main.cpp:19:
/nix/store/digm4j5579srs2xg8g8zgv7a0lh9zkxh-hyprland-0.52.0-dev/include/hyprland/src/plugins/PluginAPI.hpp:27:10: fatal error: ../version.h: No such file or directory
   27 | #include &quot;../version.h&quot;
      |          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/xtra-dispatchers.dir/build.make:93: CMakeFiles/xtra-dispatchers.dir/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:87: CMakeFiles/xtra-dispatchers.dir/all] Error 2
make: *** [Makefile:136: all] Error 2</pre>
</details>
</details>

--- a/pkgs/applications/window-managers/hyprwm/hyprland-plugins/hyprland-plugins.nix
+++ b/pkgs/applications/window-managers/hyprwm/hyprland-plugins/hyprland-plugins.nix
@@ -13,13 +13,13 @@ let
             mkHyprlandPlugin,
           }:
           let
-            version = "0.51.0";
+            version = "0.52.0";
 
             hyprland-plugins-src = fetchFromGitHub {
               owner = "hyprwm";
               repo = "hyprland-plugins";
               tag = "v${version}";
-              hash = "sha256-6jAtMjnWq8kty/dpPbIKxIupUG+WAE2AKMIKhxdLYNo=";
+              hash = "sha256-hr53AWO96ooLCwS1a2v416eT1/aWQZmuQV0ULqhaBTY=";
             };
           in
           mkHyprlandPlugin {

--- a/pkgs/by-name/hy/hyprland/package.nix
+++ b/pkgs/by-name/hy/hyprland/package.nix
@@ -3,6 +3,7 @@
   stdenv,
   stdenvAdapters,
   fetchFromGitHub,
+  fetchpatch,
   pkg-config,
   makeWrapper,
   cmake,
@@ -100,6 +101,14 @@ customStdenv.mkDerivation (finalAttrs: {
     tag = "v${finalAttrs.version}";
     hash = "sha256-5jYD01l95U/HTfZMAccAvhSnrWgHIRWEjLi9R4wPIVI=";
   };
+
+  patches = [
+    # NOTE: this is required to make plugins compile. should be removed with the next release.
+    (fetchpatch {
+      url = "https://github.com/hyprwm/Hyprland/commit/522edc87126a48f3ce4891747b6a92a22385b1e7.patch";
+      hash = "sha256-0BAlAVW5isa8gd833PjZdqO/uEpDqdTlu0iZbLP4U9s=";
+    })
+  ];
 
   postPatch = ''
     # Fix hardcoded paths to /usr installation

